scania-digital-design-system/sdds
# feat(modal): open modal programmatically

**Describe pull-request**  
Method that opens modal programmatically

**Solving issue**  
Fixes: AB#2021

**How to test**  
1. Open Storybook link -> Modal story -> Try to open modal from console running "openModal" method. Be aware that sometimes you have to visit "Elements" tab in the inspector and then go back to "Console" to make sure the console can fetch elements.
2. Another way: Check in angular, I have added a button there with method, just make sure to link packages.
